app/services/imports/logs_import_service.rb

```diff
@@ -11,11 +11,7 @@ def safe_string_as_integer(xml_doc, attribute)
     # Unsafe: A string that has more than just the integer value
     def unsafe_string_as_integer(xml_doc, attribute)
       str = string_or_nil(xml_doc, attribute)
-      if str.nil?
-        nil
-      else
-        str.to_i
-      end
+      str&.to_i
```

Comment:
someone sense check me here, but if str is nil we get nil, otherwise we get str.to_i